v2_TO_v3_MIGRATION_GUIDE.md

```diff
@@ -301,6 +301,15 @@ In order to facilitate `bigint` as a first-class type in this SDK, additional JS
 
 If your v2 code uses `JSON.parse` or `JSON.stringify` on types which can now contain `bigint`s in v3, you may receive an error such as `TypeError: Do not know how to serialize a BigInt`. Consider using these new functions instead. Or, if the types are `Encodable`, use the new `encodeJSON` and `decodeJSON` functions described in the [Object Encoding and Decoding](#object-encoding-and-decoding) section.
 
+### Msgpack Operations
+
+The functions `encodeObj` and `decodeObj`, used to encode and decode msgpack objects, have been deprecated in v3 in favor of new functions, `msgpackRawEncode` and `msgpackRawDecode`. These functions have clearer names and differ slightly from the old functions. Specifically:
```

Comment:
These functions are being renamed in this PR, just noting that if we land on not renaming / deprecating simultaneously that this documentation needs to be updated.